src/catalog/src/system_schema/memory_table.rs

```diff
@@ -194,9 +196,8 @@ mod tests {
 
         let table = MemoryTable::new(42, "test", schema.clone(), vec![]);
 
-        assert_eq!(42, table.table_id());
-        assert_eq!("test", table.table_name());
-        assert_eq!(schema, InformationTable::schema(&table));
```

Comment:
Why you chance these test cases?

Reply:
Just some stuff introduced while refactoring, we could just keep it identical with the original one.

Reply:
These lines are moved, not removal. GitHub diff sucks.